swap_meet/item.py

```diff
@@ -1,2 +1,32 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, id = None, condition = 0):
+        if not id:
+            id = int(uuid.uuid4())
+        self.id = id
+        self.condition = condition
+
+    def condition_description(self):
+        if self.condition == 0:
+            return 'this is Garbage!'
+        elif 1 <= self.condition < 2:
```

Comment:
Careful about statements like this. They sometimes have unexpected behavior. Prefer writing out the logical expression:

```python
elif 1 <= self.condition and self.condition < 2:
```

Also, worth noting: what happens if `self.condition` is between 0 and 1?